src/main/java/agents/leeroy/Leeroy.java

```diff
@@ -35,9 +35,9 @@ public A computeNextAction(G game, long computationTime, TimeUnit timeUnit) {
         super.setTimers(computationTime, timeUnit);
         log.info("Computing action");
         Risk risk = (Risk) game;
-        setPhase(game.getBoard());
         A nextAction;
         try {
+            setPhase(game.getBoard());
```

Comment:
Was this the reason for the strange error?

Reply:
I don't think so. The strange error might still be caused by my fortification logic, but I'm not sure. In the last few runs I didn't get any error